app/controllers/locations_controller.rb

```diff
@@ -70,6 +74,8 @@ def upload_locations_csv
     else
       @parent_organisation = current_organisation
       BulkUpload::BulkUpload.add_location_data(@upload_form.data, @parent_organisation)
+      @locations = @parent_organisation.locations.select(&:new_record?)
```

Comment:
Could this variable be a bit more descriptive? Like `@new_locations` or `@added_locations` or something so we know it is not just all locations - same with the `@ip_count`